apiary.apib

```diff
@@ -3262,6 +3265,12 @@ Create a new milestone.
         + due_on: `2018-01-01` (string, required)
         + name: `Initial setup` (string, required)
         + responsible_user_id: `e1240972-6cfc-4549-b49c-edda7568cc48` (string, required)
+        + `invoicing_method`: `time_and_materials` (enum[string])
+            + Members
+                + time_and_materials
+                + fixed_price
+                + non_invoiceable
+        + `invoicing_amount`: `599.50` (number, optional) - Required when method is `fixed_price`.
```

Comment:
Q: what happens when you provide an `invoicing_amount`, and the `invoicing_method` is *not* `fixed_price`? Maybe we should add "If the method is anything else, this value is ignored" or something?

Reply:
> Q: what happens when you provide an `invoicing_amount`, and the `invoicing_method` is _not_ `fixed_price`? Maybe we should add "If the method is anything else, this value is ignored" or something?

Correct, the amount is ignored if method is not fixed_price. will update